### PR TITLE
tidb-in-kubernetes: add one faq item

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR adds one faq in `faq.md` to explain the scheduling failure of Pods, and update `tidb-operator-overview.md` accordingly.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1890
https://github.com/pingcap/docs-cn/pull/1973

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

**Comment by lilin90** [comment]
@anotherrachel Is this still a draft?

**Comment by anotherrachel** [comment]
@TomShawn @DanielZhangQD PTAL, thanks!

File: dev/tidb-in-kubernetes/faq.md [hunk 1]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
...
     kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
     ```
+
+## Why Pods are not scheduled when I deploy the TiDB clusters?
+
+The reasons why TiDB Operator failed to schedule a Pod include:
+
+* The Pod is in the `Pending` state, which caused by insufficient resources or HA Policy. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).
+
+* The `taint` property are applied to some Nodes, which refuses the Pod to be scheduled unless that Pod has a matching `toleration`. Refer to the details in [taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
* The `taint` property is applied to some Nodes, which refuses the Pod to be scheduled unless that Pod has a matching `toleration`. Refer to the details in [taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).
```

File: dev/tidb-in-kubernetes/faq.md [hunk 2]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
...
+
+* The Pod is stuck in the `ContainerCreating` state and causes scheduling failure. In such case, you can check if there is more than one TiDB Operator deployed in the Kubernetes cluster. Conflicts occur when custom schedulers in multiple TiDB Operators schedule the same Pod in different phases.
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
* Scheduling conflict, which causes the Pod stuck in the `ContainerCreating` state. In such case, you can check if there is more than one TiDB Operator deployed in the Kubernetes cluster. Conflicts occur when custom schedulers in multiple TiDB Operators schedule the same Pod in different phases.
```

File: dev/tidb-in-kubernetes/faq.md [hunk 3]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
...
+
+    You can execute the following command to verify whether there are more than one TiDB Operator deployed. If more than one record is returned, delete extra TiDB Operator to fix scheduling failure.
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
    You can execute the following command to verify whether there are more than one TiDB Operator deployed. If more than one record is returned, delete extra TiDB Operator to fix scheduling conflict.
```

File: dev/tidb-in-kubernetes/faq.md [hunk 4]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
     ```shell
     kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
     ```
+
+## Why Pods are not scheduled when I deploy the TiDB clusters?
+
+The reasons why TiDB Operator failed to schedule a Pod include:
+
+* The Pod is in the `Pending` state, which caused by insufficient resources or HA Policy. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
* Insufficient resources or HA Policy, causing the Pod in the `Pending` state. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).
```

File: dev/tidb-in-kubernetes/faq.md [hunk 5]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
...
+
+* The `taint` property is applied to some Nodes, which refuses the Pod to be scheduled unless that Pod has a matching `toleration`. Refer to the details in [taint & toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/).
+
+* Scheduling conflict, which causes the Pod stuck in the `ContainerCreating` state. In such case, you can check if there is more than one TiDB Operator deployed in the Kubernetes cluster. Conflicts occur when custom schedulers in multiple TiDB Operators schedule the same Pod in different phases.
+
+    You can execute the following command to verify whether there are more than one TiDB Operator deployed. If more than one record is returned, delete extra TiDB Operator to fix scheduling conflict.
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
    You can execute the following command to verify whether there is more than one TiDB Operator deployed. If more than one record is returned, delete the extra TiDB Operator to resolve the scheduling conflict.
```

File: dev/tidb-in-kubernetes/tidb-operator-overview.md [hunk 6]
```diff
@@ -8,6 +8,10 @@ category: reference
 
 TiDB Operator is an automatic operation system for TiDB clusters in Kubernetes. It provides a full management life-cycle for TiDB including deployment, upgrades, scaling, backup, fail-over, and configuration changes. With TiDB Operator, TiDB can run seamlessly in the Kubernetes clusters deployed on a public or private cloud.
 
+> **Note:**
+>
+> Only one TiDB Operator can be deployed in each Kubernetes cluster.
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/tidb-operator-overview.md]:
```suggestion
> You can only deploy one TiDB Operator in a Kubernetes cluster.
```

File: dev/tidb-in-kubernetes/faq.md [hunk 7]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
     ```shell
     kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
     ```
+
+## Why Pods are not scheduled when I deploy the TiDB clusters?
+
+The reasons why TiDB Operator failed to schedule a Pod include:
+
+* Insufficient resources or HA Policy, which causes the Pod in the `Pending` state. Refer to the details in [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state).
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
* Insufficient resources or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```

**Comment by anotherrachel** [comment]
@TomShawn I've addressed comments and updated v3.0, PTAL.

File: dev/tidb-in-kubernetes/faq.md [hunk 8]
```diff
@@ -77,3 +77,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
     ```shell
     kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
     ```
+
+## Why does TiDB Operator fail to schedule Pods when I deploy the TiDB clusters?
+
+Three possible reasons:
+
+* Insufficient resources or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```
Comment (TomShawn) [inline dev/tidb-in-kubernetes/faq.md]:
```suggestion
* Insufficient resource or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/dev/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```

File: v3.0/tidb-in-kubernetes/faq.md [hunk 9]
```diff
@@ -78,3 +78,21 @@ To check the configuration of the PD, TiKV, and TiDB components of the current c
     ```shell
     kubectl exec -it <tidb-pod-name> -c tidb -n <namespace> -- cat /etc/tidb/tidb.toml
     ```
+
+## Why does TiDB Operator fail to schedule Pods when I deploy the TiDB clusters?
+
+Three possible reasons:
+
+* Insufficient resources or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/v3.0/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```
Comment (TomShawn) [inline v3.0/tidb-in-kubernetes/faq.md]:
```suggestion
* Insufficient resource or HA Policy causes the Pod stuck in the `Pending` state. Refer to [Troubleshoot TiDB in Kubernetes](/v3.0/tidb-in-kubernetes/troubleshoot.md#the-pod-is-in-the-pending-state) for more details.
```

**Pull request merged**